src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -237,10 +238,14 @@ function setupVisualizers() {
 
       currentSampleGain /= (currentSamples.length * currentSamples[0].length);
 
-      drawLiveGain(currentSampleGain);
+      const liveGain = currentSampleGain * monitorNode.gain.value;
+      // "Times 5" make the visulization more clear to the users
+      drawLiveGain(liveGain * 5);
```

Comment:
I would create `WAVEFROM_SCALE_FACTOR = 5` at the top of this file with a relevant comment.

Do you know the max/min value of `liveGain`? That helps you decide the scale factor accordingly.

Reply:
That's a good point! The canvas height is 150, so I believe the max value of Y is 75. Actually, I choose 5 is based on the manual testing visualization. Based on the calculation, I can not find a good factor. Do you know any formula which might help us to identify a better value for WAVEFROM_SCALE_FACTOR?